src/sg/flybot/pullable/pattern.clj

```diff
@@ -70,5 +73,79 @@
        (pattern-error! "unable to understand" pattern)))))
 
 (comment
-  (->query #(concat % ['ok]) '{(:a :with [{:b 2 :c 3}]) {:b ?}})
+  (->query #(concat % ['ok]) '{(:a :with [{:b 2 :c 3}]) {:b ?}}) 
   )
+
+(defn valid-symbol?
+  "Returns true if `p` is a symbol starting with ?"
+  [p]
+  (and (symbol? p) (re-matches #"\?.*" (name p))))
```

Comment:
Is this a replacement of `lvar?`

Reply:
It is lvar + ?
So both `'?x` or `'?` are valid